core/src/main/scala/com/stripe/dagon/FunctionK.scala

```diff
@@ -11,3 +11,10 @@ trait FunctionK[T[_], R[_]] {
 
   def toFunction[U]: T[U] => R[U]
 }
+
+object FunctionK {
```

Comment:
Is this related?

Reply:
nope, I thought I wanted it, but I realized I can't use it as I was hoping. Would you like me to remove it?